core/src/main/java/org/merakilearn/core/navigator/MerakiNavigator.kt

```diff
@@ -66,6 +67,11 @@ class MerakiNavigator(
         startActivity(context, openRoomIntent(context, roomId), buildTask)
     }
 
+    fun openChatApp(activity: Activity) {
+        chatModuleNavigator.launchChatApp(activity)
```

Comment:
should be removed or replaced with opening chat tab in home screen

Reply:
Done